### ISSUE TITLE
Request: Add support for frame properties (Avs+)

### ISSUE THREAD

**Issue by TbtBI** [FFMS/ffms2#385]
Hi,

Can you add frame properties support for Avs+?

**Comment by realfinder** [comment]
this https://forum.doom9.org/showpost.php?p=1949799&postcount=2666 got ffms2-1_avs_properties.patch and another patches

**Comment by TbtBI** [comment]
Thanks.
This patch and probably the other one for #386 could be incorporated to the upstream, @myrsloik?

**Comment by myrsloik** [comment]
Where's the actual patch? Also, pull requests are welcome but people seem to prefer forkinating all the things...

**Comment by realfinder** [comment]
inside ffms2_1c6169a, anyway those [ffms2 patches.zip](https://github.com/FFMS/ffms2/files/6988326/ffms2.patches.zip)

StvG seems don't want to use github

**Comment by myrsloik** [comment]
Done, a fixed version of the patch has been applied. It probably produced garbage _AbsoluteTime values in some cases and a few other oddities.

**Comment by TbtBI** [comment]
Thanks for the fast implementation.

Can the same be done with drc_scale patch/arg?

I know you are aware of [this doom9 post](https://forum.doom9.org/showthread.php?p=1921568#post1921568). StvG builds have reverted back commits and they are using the old way of how ffms2 works, and that's why his other patches cannot be incorporated, right? Cannot something be done to eliminate the regression of the upstream builds compared to StvG builds? From the seeking tests I saw in doom9 (I tested with few other samples too) between the StvG builds and the official one, StvG builds are at least good as the official one. It would be great if the upstream builds are at least good as that custom builds.
